Sources/SwiftSyntaxBuilder/TupleExprElementConvenienceInitializers.swift

```diff
@@ -0,0 +1,22 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+public extension TupleExprElement {
+  /// A convenience initializer that allows passing in label as an optional string.
+  /// The presence of the colon will be inferred based on the presence of the label.
+  init(_ label: String? = nil, expression: ExpressibleAsExprBuildable) {
```

Comment:
I don’t have super strong opinions but I think I would prefer `label` to be a labelled argument instead of an unlabelled one. What was your reasoning for making it unlabeled?

Reply:
Just wanted to make the signature different to the existing one, but I'm fine to make it labelled